renovate.json

```diff
@@ -1,3 +1,11 @@
 {
-  "extends": ["config:base", "schedule:earlyMondays"]
-}
+  "extends": [
+    "config:base",
+    "schedule:earlyMondays"
+  ],
+  "automerge": true,
```

Comment:
GRPC safe? 🤔

Reply:
Won't merge without explicit approval since that's a status check that has to pass.  Once we hit approve, it'll go through.

Reply:
👍